2/src/components/List.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+     <div id="list">
+       <list-item 
+                v-for="todo in todos"            
+                :key="todo.id"
+                :todo="todo"
+                @update-todo="updateTodo"
+                @delete-todo="deleteTodo">
+        </list-item>
+      </div>
+</template>
+
+<script>
+import ListItem from "./ListItem.vue";
+export default
+{   name: 'list',
+    props: ['todos'],
```

Comment:
```suggestion
    props: ['todos'], props: ["todo", "index"]
```
As I see it, you could add an "index" to the "props", which you could use later in deleteTodo: function(id)

Reply:
But this is `List.vue` not `ListItem.vue`, isn't it? :thinking: